File: leetcode-top-interview-150/1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(len(nums) - 1):
            answer = target - nums[i]

            for j in range(len(nums) - 1, i, -1):
                if nums[j] == answer:
                    return [i, j]

File: leetcode-top-100-liked/1.kt
class Solution {
    fun twoSum(nums: IntArray, target: Int): IntArray {
        val map: MutableMap<Int, Int> = mutableMapOf()

        for(i in nums.indices) {
            var curVal = nums[i]
            if(map.contains(target - curVal)){
                return intArrayOf(i, map[target-curVal]!!)
            }

            map[curVal] = i
        }

        return intArrayOf(1,1)

    }
}